src/components/pages/account-page/types/index.ts

```diff
@@ -0,0 +1,13 @@
+export enum AccountPageTab {
+  GENERAL = 'general',
+  SECURITY = 'security',
+  GROUP = 'group',
+  SELECTIVE = 'selective',
+}
+
+export enum AccountPagesMapper {
+  group = 'Група',
+  security = 'Безпека',
+  general = 'Загальне',
+  selective = 'Мої вибіркові',
+}
```

Comment:
Due to enum naming conventions, as you did above, make keys with upper case too.